src/main/resources/db/migration/V1_5__add_expired_columns_to_varsel_table.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE varsel
+    ADD COLUMN expires_at TIMESTAMPTZ DEFAULT NULL;
+
+ALTER TABLE varsel
+    ADD COLUMN is_expiration_published BOOLEAN DEFAULT FALSE;
```

Comment:
Synes godt denne kan være `TIMESTAMPZ` også (typ expiration_published_at) så fungerer den både som flagg og med informasjon om når det skjedde i tilfelle vi skulle trenge det.

Reply:
Allright! Domenemessig tenkte jeg det var rar/unødvendig informasjon å ha, men kan jo bli nyttig down the road. Domeneobjektet kan jo uansett representere det som `Boolean` om vi ønsker det